stagecraft/apps/organisation/tests/test_views.py

```diff
@@ -372,13 +365,6 @@ def setUp(self):
         )
         brie.parents.add(cheese)
 
-    def test_root_view_doesnt_delete(self):
-        delete_resp = self.client.delete(
-            '/organisation/type',
-            HTTP_AUTHORIZATION='Bearer development-oauth-access-token')
```

Comment:
I suppose it's probably the correct behaviour but deleting a type will cascade and delete the nodes I think. Just a heads up about the risks of enabling deletion on everything.